partiql-tests-data/eval/primitives/operators/bag-operators.ion

```diff
@@ -143,56 +143,204 @@ bagOperators::[
       ]
     }
   },
+  // outer union coercion
```

Comment:
Tests from L146-346 do the following
- For strict mode, give an error when coercing a non-collection
- Adds `INTERSECT` and `EXCEPT` versions of the existing `OUTER UNION` tests